R/simpleheatmap3.R

```diff
@@ -66,6 +73,27 @@ simpleheatmap3 <- function(pln,
                                  clusterID = clusterIDsCol,
                                  stringsAsFactors = F)))
   }
+  else if(plot&suppressColSideCols) {
```

Comment:
Please leave white spaces between operator plot & suppressColSideCols
```suggestion
  else if(plot & suppressColSideCols) {
```